bookmarks/styles/bookmarks.scss

```diff
@@ -58,6 +58,10 @@ ul.bookmark-list {
     text-overflow: ellipsis;
   }
 
+  .url-display {
+    color: $green-color;
```

Comment:
Not happy with adding another color to the palette, the current green feels very "off". Maybe we could just lighten / darken the default link color somewhat.

From some testing:
- Light theme: rgba(87, 85, 217, 0.64)
- Dark theme: rgba(168, 177, 255, 0.73)

Reply:
Renamed variable to` $secondary-link-color` and added palette colors to respective scss files for dark and light themes.